app/index.ts

```diff
@@ -17,33 +17,31 @@ const SETTINGS_FILE = "settings.json";
 let settings = loadSettings();
 applyTheme(settings.foreground);
 
-function updateClock(evt) {
+clock.ontick = evt => {
   let today = evt.date;
-  let hours = today.getHours();
-  if (preferences.clockDisplay === "12h") {
-    // 12h format
-    hours = (hours + 24) % 12 || 12;
-  }
-  else {
-    // 24h format
-    hours = util.zeroPad(hours);
```

Comment:
`hours` is initialized as a `number`, but here assigned to a `string`. And TypeScript complained about that.